src/python/chrome/build_info.py

```diff
@@ -43,6 +45,15 @@ def _convert_platform_to_omahaproxy_platform(platform):
   return platform_lower
 
 
+def _convert_platform_to_chromiumdash_platform(platform):
+  """Converts platform to Chromium Dash platform.
+  Note that Windows in Chromium Dash is win64 and we only want win32."""
+  platform_lower = platform.lower()
+  if platform_lower in ('windows', 'win'):
```

Comment:
why do we need to compare against 'win'? Shouldn't be needed?

Reply:
Omahaproxy uses `win` while ChromiumDash use `Win32`. And in `get_release_milestone()`, it passes `win` as the platform. Not sure if anywhere in the code/config files also use this format. So I convert both formats here.

Reply:
I see, that's a bit confusing!

In that case let's fix `get_release_milestone()` to pass 'windows' instead. Everywhere else in ClusterFuzz, we generally use 'windows'. Nowhere else should be passing 'win'.

Reply:
Done